WooCommerce/Classes/ViewRelated/Orders/Order Details/Receipts/ReceiptViewController.swift

```diff
@@ -0,0 +1,72 @@
+import WebKit
+
+/// Previews a backend-generated receipt
+final class ReceiptViewController: UIViewController {
+    @IBOutlet private weak var webView: WKWebView!
+    private let viewModel: ReceiptViewModel
+
+    init(viewModel: ReceiptViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: "LegacyReceiptViewController", bundle: nil)
```

Comment:
Since the nib is exactly the same as the one used for `LegacyReceiptViewController` I opted to use the same one here, when we remove the legacy receipt we can update this and keep the existing nib.

Reply:
No worries about cleaning up later, but it might be clearer to rename `LegacyReceiptViewController.xib` back to `ReceiptViewController.xib`, then you can use the nibName forClass stuff here. You'll have to specify it with the string in `LegacyReceiptViewController.swift`, but that's the code which is going away anyway.